### ISSUE TITLE
Some actions are hard to achieve

### ISSUE THREAD

**Issue by Zwatotem** [FancyWM/fancywm-issues#75]
**Describe the bug**
I cannot move a window to the other side across a vertical panel

**To Reproduce**
1. Open open four windows, best for this demo if they have low minimal widths.
2. Create a vertical panel on the rightmost window.
3. Add any window to that vertical panel.
Window layout should look something like this:
![image](https://user-images.githubusercontent.com/24753312/161917758-b0a3da2e-ffa4-4965-9b13-439879b2b00d.png)
4. Now try to move the middle window with a mouse to the right, so that the vertical panel is in the middle.
5. The moved window snaps only within the vertical panel, not letting me place it past it.
6. Try to use a hotkey to move the window.
7. You can move window into the panel, but no further (I get the message: "**Could not move right!**  No adjacent window!")
8. Try to move the panel to the left.
9. Finally achieved the expected result, but didn't get any visual feedback animation, that this going to work.

**Expected behavior**
While I initially thought this is maneuver is impossible I find out now that it is possible. However the first method should work as well, as it's the natural way of attempting this. Overall I think where the window is going to be placed should be determined by regions-between-windows, where it's being hovered over, rather than what window it is being held over (how I assume it is handled now).

**Screenshots**
![image](https://user-images.githubusercontent.com/24753312/161917758-b0a3da2e-ffa4-4965-9b13-439879b2b00d.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version: 22H2 22581.200 64-bit

**Additional context**

**Comment by veselink1** [comment]
Thanks for reporting this. I agree the current behaviour can be improved.

You are correct that the final window position is based on the position of the mouse cursor on-screen. While it is possible to use the window region to guide the reordering, my initial experiments showed that was less intuitive. I plan to go back to the drawing board at some point and rethink those decisions at some point. 

As a side-note: It is possible to move the window to the right of the panel by grabbing the window and moving the mouse cursor along the top of the screen so that it doesn't intersect with the regions occupied by the windows in the vertical panel. This is non-ideal but is another way to achieve the same effect.

**Comment by mataslib** [comment]
I agree as I'm also often fighting with switching positions of vertical panel with normal window.

**Comment by jweickm** [comment]
> As a side-note: It is possible to move the window to the right of the panel by grabbing the window and moving the mouse cursor along the top of the screen so that it doesn't intersect with the regions occupied by the windows in the vertical panel. This is non-ideal but is another way to achieve the same effect.

Hi, it would be great if this could be added to the FAQ, as I was also struggling with this and just found out about this (workable) solution reading this thread.
